ui/components/app/smart-transactions-modal/smart-transactions-modal.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  TextColor,
+  Display,
+  FlexDirection,
+  FontWeight,
+  BlockSize,
+  AlignItems,
+  JustifyContent,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  Modal,
+  ModalOverlay,
+  Text,
+  Box,
+  Button,
+  ButtonVariant,
+  ModalHeader,
+  ModalContent,
+} from '../../component-library';
+
+type Props = {
```

Comment:
I believe the standard is to export the props type so it could be called `SmartTransactionsModalProps`?